ci/unittests.yml

```diff
@@ -22,7 +22,7 @@ dependencies:
   - pytest-xdist
   - setuptools
   - xarray
-  - xarray-datatree
   - xmltodict
   - pip:
     - git+https://github.com/openradar/open-radar-data.git
+    - git+https://github.com/xarray-contrib/datatree
```

Comment:
I'm assuming there is bleeding edge code used from datatree, right?

Reply:
Yeah - I opened an issue over at datatree requesting a new release